### DISCUSSION TITLE
Inline images

### DISCUSSION THREAD

**Discussion by johnmapeson** [typst/typst#4922] [Q&A] [answered]
Does typst support inline images? That is, to get the output like this one:

```
Lorem ipsum <image> dolor sit amet.
```

I tried

```
Lorem ipsum #image("image.png", width: 0.1in) dolor sit amet.
```

but then the output looks differently:

```
Lorem ipsum

<image>

dolor sit amet.
```

**Comment by laurmaedje** [comment, marked as answer]
Most things are block-level. You can turn anything block-level inline with [`box`](https://typst.app/docs/reference/layout/box/).
